stable/superset/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 description: Apache Superset (incubating) is a modern, enterprise-ready business intelligence web application
 name: superset
-version: 1.0.0
-appVersion: "0.24.0"
+version: 1.0.1
+appVersion: "0.28.1"
```

Comment:
please update the readme.md file as well

Reply:
and bump the chart version to `1.1.0`

Reply:
Done!